adhocracy4/polls/models.py

```diff
@@ -157,12 +166,11 @@ def __str__(self):
         return '%s: %s' % (self.creator, self.choice)
 
 
-class OtherVote(UserGeneratedContentModel):
-    question = models.ForeignKey(
-        'Question',
-        on_delete=models.CASCADE,
-        related_name='other_votes',
-    )
+class OtherVote(models.Model):
+    vote = models.OneToOneField(Vote,
+                                on_delete=models.CASCADE,
+                                primary_key=True,
+                                related_name='other_vote')
```

Comment:
I like the other markup more and we use that in quite a few places, but I guess that's Geschmackssache. :)